### ISSUE TITLE
agitator fails on CentOS 7.9

### ISSUE THREAD

**Issue by dlmarion** [apache/accumulo-testing#237]
Error in the agitator logs:

```
./agitator: line 44: readarray: -d: invalid option
readarray: usage: readarray [-n count] [-O origin] [-s count] [-t] [-u fd] [-C callback] [-c quantum] [array]
ERROR: No hosts were found in env for tserver
```

Running with bash `GNU bash, version 4.2.46(2)-release (x86_64-redhat-linux-gnu)` on `CentOS Linux release 7.9.2009 (Core)`

**Comment by dlmarion** [comment]
Looks like -d was added in bash 4.4

**Comment by DomGarguilo** [comment]
> Looks like -d was added in bash 4.4

I wonder if we should somehow warn the user that this is the case (or require bash >= 4.4) instead of letting the error you saw occur.

**Comment by DomGarguilo** [comment]
It appears the agitator is working for me with the following:
bash version: `GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)`
OS: `Pop!_OS 22.04 LTS`

**Comment by ctubbsii** [comment]
This should be fixed in my PR #238, but I have not tested it
